.github/workflows/sync-from-documentation-to-main.yml

```
@@ -0,0 +1,49 @@
+name: Sync from documentation to main
+
+permissions:
+  contents: write
+
+on:
+  push:
+    branches:
+      - documentation-content
+    paths:
+      - documentation/content/**
+      - documentation/public/admin/images/**
+
+jobs:
+  sync:
+    if: "startsWith(github.event.head_commit.message, '[NetlifyCMS]')"
+    name: Sync only the documentation content relevant folders from documentation-content into the main branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          ref: main
+          fetch-depth: 0
```

```suggestion
      - uses: actions/checkout@v3
```
Suggestion: use `v3` instead of `master` so it's not using unreleased, potentially broken code

Suggestion: removed `fetch-depth: 0` is the default. 

Suggestion: `ref` defaults to the commit that triggered the run e.g. the commit just pushed. This is (probably) preferable to `main` as:
1. I think `tj-actions/changed-files` works best if the actual commit is checked out
2. The specific commit is better than a branch name (e.g. `documentation-content`) as it prevents race conditions e.g. multiple changes in quick succession